ngraph/core/src/op/split.cpp

```diff
@@ -196,20 +194,25 @@ shared_ptr<Node> op::v1::Split::clone_with_new_inputs(const OutputVector& new_ar
 
 namespace
 {
-    inline bool evaluate(const HostTensorPtr& in,
-                         const HostTensorPtr& out,
-                         const Coordinate& lower_bounds,
-                         const Coordinate& upper_bounds)
+    inline bool evaluate(const HostTensorPtr& data_tensor,
+                         const HostTensorVector& outputs,
+                         const int64_t axis,
+                         const int64_t num_splits)
     {
-        runtime::reference::slice(in->get_data_ptr<const char>(),
-                                  out->get_data_ptr<char>(),
-                                  in->get_shape(),
-                                  lower_bounds,
-                                  upper_bounds,
-                                  Strides(lower_bounds.size(), 1),
-                                  out->get_shape(),
-                                  in->get_element_type().size());
-
+        Shape output_shape = data_tensor->get_shape();
+        std::vector<char*> outputs_data(num_splits);
+        output_shape.at(axis) /= num_splits;
+        for (size_t i = 0; i < outputs.size(); ++i)
+        {
+            outputs[i]->set_shape(output_shape);
+            outputs_data[i] = outputs[i]->get_data_ptr<char>();
+        }
+        ngraph::runtime::reference::split(data_tensor->get_data_ptr<char>(),
```

Comment:
Why you changed slice to split?

Reply:
split is placed in a separate reference implementation, it's a convenient wrapper for a slice